cmd/main.go

```diff
@@ -30,10 +30,10 @@ import (
 )
 
 func main() {
-	idx := slices.Index(os.Args, "--run-with-subprocess")
+	idx := slices.Index(os.Args, "--run-without-subprocess")
 
-	// execute command as a subprocess if the command contains "--run-with-subprocess"
-	if idx > 0 {
+	// execute command as a subprocess if --run-without-subprocess is not sets
+	if idx < 0 {
 		args := slices.Delete(os.Args, idx, idx+1)
```

Comment:
Delete arg shall happen when param `run-without-subprocess` is provided

Reply:
Yes,  this opt should not pass into milvus arguments